io.elementary.capnet-assist.yml

```
@@ -0,0 +1,27 @@
+app-id: io.elementary.capnet-assist
+
+runtime: io.elementary.Platform
+runtime-version: 'daily'
+sdk: io.elementary.Sdk
+
+command: io.elementary.capnet-assist
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+
```

don't forget the sandbox hole for the dark style preference :)
```suggestion
  - '--system-talk-name=org.freedesktop.Accounts'
```

---

Oh man, is this mentioned in the developer guide and I totally missed it?

---

I actually just added a section for it the other day! So it probably wasn't when you pushed this